### PR TITLE
Adding chart titles

### PR DESCRIPTION
**Pull request by bitamar**
#700

### CODE REVIEW

**Comment by bitamar** [comment]
screen shot 2015-07-21 at 10 48 35 am
screen shot 2015-07-21 at 10 48 14 am

**Comment by bitamar** [comment]
screen shot 2015-07-21 at 12 01 53 pm

File: client/app/scripts/directives/state-title.directive.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * @description
+ *   Display the title of the selected marker, or category when there's no
+ *   marker.
+ */
+angular.module('negawattClientApp')
+  .directive('stateTitle', function () {
+    return {
+      template: '<h4 class="state-title title">{{ title }}</h4>',
+
+      controller: function($scope, $stateParams, Meter, Category, Profile) {
```
Comment (amitaibu) [inline client/app/scripts/directives/state-title.directive.js]: don't pass $stateParams

File: client/app/scripts/directives/state-title.directive.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+/**
+ * @ngdoc directive
+ * @name negawattClientApp.directive:stateTitle
+ * @description
+ *   Display the title of the selected marker, or category when there's no
+ *   marker.
+ */
+angular.module('negawattClientApp')
+  .directive('stateTitle', function () {
```
Comment (amitaibu) [inline client/app/scripts/directives/state-title.directive.js]: `<state-title="{title}"></state-title>`

**Comment by bitamar** [comment]
Merged

**Pull request closed without merging**